host_applications/linux/apps/raspicam/RaspiCamControl.c

```diff
@@ -214,6 +215,7 @@ static COMMAND_LIST  cmdline_commands[] =
    {CommandAnalogGain,  "-analoggain", "ag", "Set the analog gain (floating point)", 1},
    {CommandDigitalGain, "-digitalgain", "dg", "Set the digital gain (floating point)", 1},
    {CommandSettings,    "-settings",   "set","Retrieve camera settings and write to stdout", 0},
+   {CommandFocusWindow, "-focus",      "fw","Display a window with the focus FoM value on an overlay.", 0},
```

Comment:
MMAL_PARAMETER_DRAW_BOX_FACES_AND_FOCUS draws the focus box on the camera image, not as an overlay.

Otherwise the patch is fine.